.drone.yml

```diff
@@ -0,0 +1,85 @@
+workspace:
+  base: /go
+  path: src/github.com/src-d/landing
+
+branches: master
+
+pipeline:
+
+  build:
+    image: golang:1.8
+    commands:
+      - apt-get update
+      - apt-get install -y make bash curl git
+      - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.4/install.sh | sh
+      - . ~/.nvm/nvm.sh
+      - nvm install 8
+      - npm install -g yarn
```

Comment:
The [travis file did `nvm use 8`](https://github.com/src-d/landing/pull/159/files#diff-354f30a63fb0907d4ad57269548329e3L29); is it not needed at all?

Reply:
Well, I since the container is a freshly installed system and we just installed v8 and no other, I don't think it's necessary, but I can include it, just in case.